third-culture-software/bhima
# Convert cost allocation basis units to translation token for predefined bases

Converts the 'units' field cost allocation basis items to be a translation token for predefined basis items so that they can be translated when possible.

Cleaned up/reorganized many of the cost allocation translation items into cost_center.json.  Deleted the old fee_center.json files (not needed after the reorganization).  Reworded some of the terms related to cost allocations.

Temporarily disabled the display of the  old cost distribution pages.  No code was deleted *yet*.

Closes https://github.com/IMA-WorldHealth/bhima/issues/5926

Partially implements https://github.com/IMA-WorldHealth/bhima/issues/5926: the cost allocation basis table is modified with the new data items but the formatting of the tabular display was not done yet.

*TESTING*
- Use bhima_test
- yarn; yarn build; yarn build:db
- Load the test data in server/models/migration/next/sample_cost_center_data.sql
- Try the functions under the Cost Center Management page and notice that the units for predefined basis items work properly (in English and French).
- Please review the French translations!
